src/renderer/tvgFill.h

```diff
@@ -128,7 +128,41 @@ struct RadialGradientImpl : RadialGradient
         if (fy) *fy = this->fy;
         if (fr) *fr = this->fr;
 
-        return Result::Success;
+        //TODO: remove the "internal" logic once SVG 2.0 is adopted by sw and wg engines (gl already supports it); lottie-specific handling will then be delegated entirely to the loader
+        if (!internal) return true;
+
+        constexpr float precision = 0.01f;
+
+        //solid fill - handled internaly in engines
```

Comment:
Please use like this, Renderer guides a spec to engine. It has no idea about each engine.
`//a solid fill case. It can be handled by engine.`